demisto_sdk/commands/validate/README.md

```diff
@@ -26,10 +26,11 @@ Create the id_set.json file.
 * **--prev-ver**
 Previous branch or SHA1 commit to run checks against.
 * **-g, --use-git**
-Validate changes using git - this will check your branch changes and will run only on them.
+Validate changes using git - this will check the current branch's changes against origin/master and will run only on them.
+If **--post-commit** is supplied: validation will run only on committed branch files, if not: validation will run on both committed and non-committed branch changed files
 * **--post-commit**
-Whether the validation is done after you committed your files, this will help the command to determine which files it
- should check in its run. Before you commit the files it should not be used. Mostly for build validations.
+Whether the validation should run only on current branch's committed changed files.
+This applies only when **-g** flag is supplied
```

Comment:
Change the `README` descriptions to match the suggestions I made for the `__main__.py`